ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -173,7 +174,7 @@ function WorkloadCvesOverviewPage() {
 
     const currentVulnerabilityState = useVulnerabilityState();
 
-    const { searchFilter, setSearchFilter } = useURLSearch();
+    const { searchFilter, setSearchFilter: setURLSearchFilter } = useURLSearch();
```

Comment:
We potentially need to update the sort every time the filter on the page changes. The `setSearchFilter` function is aliased here, and a replacement `setSearchFilter` calls the aliased function and then syncs the sort parameters in order to create a single mutation point and avoid a large number of changes.